README.Rmd

```diff
@@ -0,0 +1,105 @@
+---
+output: github_document
+---
+
+<!-- README.md is generated from README.Rmd. Please edit that file -->
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.path = "man/figures/README-",
+  out.width = "75%",
+  fig.align = "center"
+)
+```
+
+# _{{ packagename }}_ <img src="man/figures/logo.png" align="right" width="120" />
+
+<!-- badges: start -->
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+[![R-CMD-check](https://github.com/epiverse-trace/epiparameter/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/epiverse-trace/epiparameter/actions/workflows/R-CMD-check.yaml)
+[![Codecov test coverage](https://codecov.io/gh/epiverse-trace/epiparameter/branch/main/graph/badge.svg)](https://app.codecov.io/gh/epiverse-trace/epiparameter?branch=main)
+[![Lifecycle: experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental)
+<!-- badges: end -->
+
+_{{ packagename }}_ is an `R` package that contains a library of epidemiological parameters for infectious diseases and a set classes and helper functions to be able to work with the data. It also includes functions to extract and convert parameters from reported summary statistics.
+
+_{{ packagename }}_ is developed at the [Centre for the Mathematical Modelling of Infectious Diseases](https://www.lshtm.ac.uk/research/centres/centre-mathematical-modelling-infectious-diseases) at the [London School of Hygiene and Tropical Medicine](https://www.lshtm.ac.uk/) as part of [Epiverse-TRACE](https://data.org/initiatives/epiverse/).
+
+
+## Installation
+
+The easiest way to install the development version of _{{ packagename }}_ is to use the `{pak}` package:
+
+``` r
+# check whether {pak} is installed
+if(!require("pak")) install.packages("pak")
+pak::pak("{{ gh_repo }}")
+```
+
+## Quick start
+
+```{r load-epiparameter}
+library(epiparameter)
+```
+
+To load the library of epidemiological parameters into `R`:
+
+```{r load-data}
+eparams <- epiparam()
+eparams
+```
+
+The library is an `<epiparam>` class, which underneath is a data frame.
+
```

Comment:
Reading this from a user perspective I think that mentioning these different object classes could be confusing and not really important or significant for an external user. It would be necessary to either remove the text about the classes altogether or, if it's kept, add an explanation of what these classes are and why they are converted, which I don't know if this fits within a quick start section.

I was speaking with @jamesmbaazam about design vignettes and I think that this information about the object classes would be more fitting for this type of vignette rather than the Readme

Reply:
Thanks for raising this point. It's definitely an aspect that might confuse newer users that are unfamiliar with classes. I will reference this in an issue and make the change in an upcoming pull request.